packages/perseus-score/src/util/answer-types.ts

```diff
@@ -563,50 +563,63 @@ const KhanAnswerTypes = {
                     guess: guess,
                 };
 
-                // iterate over all the acceptable forms, and if one of the
-                // answers is correct, return true
-                acceptableForms.forEach((form) => {
```

Comment:
This feels like it should be `acceptableForms.find()`.  We're searching through `acceptableForms` and the first one that matches we're returning `true` (ie. FOUND!). 

It's a bit weird in that we're updating the `score` as we loop but once we find a match, we'd update the score and returning `true` to break out of the search.

Reply:
Yeah, I thought about using something like that too. I just feel like using something like `find` is semantically incorrect since this is mutating external variables.

I also thought about updating the logic to be something like:

```
const score = acceptableForms.reduce(
    () => { /* ...whatever... */},
    {
        empty: guess === "",
        correct: false,
        message: null as string | null | undefined,
        guess: guess,
    }
)
```

But I kind of feel like I'm walking on eggshells in this code.